server/www/templates/posts/detail.html

```diff
@@ -65,11 +65,13 @@ <h1 class="postcard-title">
             ng-if="form_attributes[key].type === 'media'"
             label="{{form_attributes[key].label}}"
             media-id="value"></post-media-value>
+
         </div>
         <div
           ng-if="post.content"
           class="postcard-field">
           <p sd-model-to-html="post.content"></p>
+          <post-preview-media post="post"></post-preview-media>
```

Comment:
This shows the media item twice, once here, and once in the `post-media-value` item above. Was that intentional??